config/config.sample.json

```diff
@@ -8,7 +8,8 @@
         "ResponseHeaderTimeoutSecs": 30,
         "ReadTimeoutSecs": 30,
         "WriteTimeoutSecs": 30,
-        "IdleTimeoutSecs": 30
+        "IdleTimeoutSecs": 30,
+        "ReverseAddressLookupValidation": false
```

Comment:
Suggest changing it to `RequestValidation` in case we want to add more logic later.